dist/lob-api-bundled.yml

```diff
@@ -3076,7 +3076,7 @@ components:
     metadata:
       in: query
       name: metadata
-      description: Filter by metadata key-value pair, e.g. `metadata[customer_id]=987654`.
+      description: 'Filter by metadata key-value pair, e.g. `metadata[customer_id]=987654`.'
```

Comment:
Just curious @maelingmurphy you aren't adding these single quote `'` strings manually right?
The bundler is doing it?

https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml